services/server/server.go

```diff
@@ -418,8 +426,9 @@ func LoadPlugins(ctx context.Context, config *srvconfig.Config) ([]*plugin.Regis
 
 			path := filepath.Join(ic.Root, "meta.db")
 			ic.Meta.Exports["path"] = path
-
-			db, err := bolt.Open(path, 0644, nil)
+			options := *bolt.DefaultOptions
+			options.Timeout = timeout.Get(boltOpenTimeout)
+			db, err := bolt.Open(path, 0644, &options)
```

Comment:
```suggestion
			doneCh := make(chan struct{}, 1)
			go func() {
				select {
				case <-time.After(30 * time.Second):
				case <-doneCh:
					log.G(ctx).WithField("plugin", "bolt").Warn("waiting for response from boltdb open")
					return
				}
			}()
			db, err := bolt.Open(path, 0644, &options)
			doneCh <- struct{}{}
```
@cpuguy83 pls take review, do u mean log like this?

Reply:
More or less.
`time.After` replaced with `time.NewTimer`, possibly with a lower timeout to make the log.
I'd also typically close the channel instead of sending on it.

Reply:
Thanks for review! does the updated `suggest change` looks good to u? 

> time.After replaced with time.NewTimer

Is this kind of code of conduct?

Reply:
A time.After can't be cancelled so it has to run the full time specified.

Reply:
So we need `t.Reset(0)` in `doneCh` case?

Reply:
Just a defer t.Stop()

Reply:
Done